app/services/bulk_tagging/document_type_tagger.rb

```diff
@@ -0,0 +1,27 @@
+module BulkTagging
+  class DocumentTypeTagger
+    def self.call(taxon_base_path:, document_type:)
+      new(taxon_base_path: taxon_base_path, document_type: document_type).call
+    end
+
+    def initialize(taxon_base_path:, document_type:)
+      @taxon_base_path = taxon_base_path
+      @document_type = document_type
+    end
+
+    def call
+      taxon_content_id = Services.publishing_api.lookup_content_id(base_path: @taxon_base_path)
+      raise StandardError, "Cannot find taxon with base path #{@taxon_base_path}" if taxon_content_id.nil?
+
+      Services.publishing_api.get_content_items_enum(document_type: @document_type, fields: ['content_id']).lazy.map do |document|
+        begin
+          content_id = document.fetch('content_id')
+          Services.publishing_api.patch_links(content_id, links: { taxons: [taxon_content_id] })
```

Comment:
This is going to eliminate all the other tags for this content, which might be ok for some uses I guess, but maybe not all...?

Reply:
I see that this has changed now, I think that is good 👍